docs/content/add-plugins/sendgrid.mdx

```diff
@@ -0,0 +1,3967 @@
+# SendGrid
+
+In this document, you’ll learn about the [SendGrid plugin](https://github.com/medusajs/medusa/tree/master/packages/medusa-plugin-sendgrid), what it does, and how to use it.
+
+<div>
+  <video width="100%" height="100%" playsinline autoplay muted controls>
+    <source src="https://user-images.githubusercontent.com/59018053/154807282-1e72671f-1936-411d-b914-e05c6597693a.mp4" type="video/mp4" />
+  </video>
+</div>
+
+## Overview
+
+[SendGrid](https://sendgrid.com) is an email and notification service that can be used to send emails to your customers and users.
+
+By integrating SendGrid with Medusa, you’ll be sending email notifications to your users in the following cases:
+
+1. Order-related events include new orders, shipments created, and orders canceled.
+2. Swaps and Returns related events including new return requests of orders and items returned successfully.
+3. When Gift Cards in an order are created.
+4. User-related events including reset passwords.
+5. Restock Notifications for when product stocks are low.
+
+## Prerequisites
+
+Before going further with this guide make sure you have a Medusa server set up. You can follow our [Quickstart guide](../quickstart/quick-start.md).
+
+You also must have [Redis configured on your Medusa server](../tutorial/0-set-up-your-development-environment.md). Sending emails is done through Subscribers, which uses Redis as the event queue. If you don’t set up Redis, the plugin will not send emails.
+
+## Create a SendGrid Account
+
+If you don’t have a SendGrid account, make sure to [create one](https://signup.sendgrid.com) first. You also need to set up a [single sender](https://docs.sendgrid.com/ui/sending-email/sender-verification) first in SendGrid before you can start sending emails.
+
+### API Key
+
+For the integration to work, you need to create an API key in your SendGrid account.
+
+You can do that by choosing from the sidebar of your SendGrid dashboard Settings > API Keys. Then, click on Create API Key.
+
+:::note
```

Comment:
question(non-blockin): Is this supposed to be here?

Reply:
Yes this adds the note formatting in docusaurus

<img width="709" alt="Screen Shot 2022-04-12 at 10 03 51 PM" src="https://user-images.githubusercontent.com/27354907/163034933-57fae476-74aa-42f2-b08e-45603b5ffbfa.png">